### ISSUE TITLE
there is no way to distinguish warning and error from the output

### ISSUE THREAD

**Issue by braindevices** [ansible/ansible-lint#2608]
##### Summary
It does not matter that I choose which output format, there is no field allow us the differentiate the warning and error.

The codeclimate/json output has a severity field but it has nothing to do with warning or error. It is the internal severity of the rule. Change rule from error to warning while `-w` does not modify the severity at all.

Please fix the output formats to let it identify what is error what is warning. Right now everything is fatal. The return code only show you there is one of the violation output is error, but you do not know which one is the actual error, this is ridiculous.

This behaviour breaks all LSP, since all of them expect the severity field give it actual severity about warning, error and fatal configured by the user.

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)
ansible --version
2.13.4
ansible-lint --version
6.7
```

- ansible installation method: one of source, pip, OS package
- ansible-lint installation method: one of source, pip, OS package

##### STEPS TO REPRODUCE

```console (paste below)

```

##### Desired Behavior

Possible security bugs should be reported via email to `security@ansible.com`

##### Actual Behavior

Please give some details of what is actually happening.
Include a [minimum complete verifiable example] with:

- playbook
- output of running ansible-lint
- if you're getting a stack trace, output of
  `ansible-playbook --syntax-check playbook`

```paste below

```

[minimum complete verifiable example]: http://stackoverflow.com/help/mcve

**Comment by ssbarnea** [comment]
The bug report form asks to test with latest version for a reason, because the problem is likely that was already addresses. The newest version does append a "(warning)" at the end of the warning lines.
